lib/GenomeFileUtil/core/GenomeInterface.py

```diff
@@ -299,19 +315,19 @@ def _update_genome(self, genome):
         # NOTE: New Taxon changes should most likely fit in here.
```

Comment:
This method is 140 lines long

Reply:
I didn't write _update_genome and I don't want to mess with it more than I have to